web/utils.go

```diff
@@ -75,7 +75,7 @@ func GetCertFiles(certFile, keyFile string) (string, string) {
 
 // GetRemoteConnection returns a connection to a remote address
 func GetRemoteConnection(useTLS bool, export *registry.ExportDetails) (remote net.Conn, err error) {
-	isLocalAddress := IsLocalAddress(export.PrivateIP)
+	isLocalAddress := IsLocalAddress(export.HostIP)
```

Comment:
That's a good question. I don't know the answer.  The existing check is cheap though, since it already caches the list of local ips in init()